### PR TITLE
Merge master into release-boba

### PR DESCRIPTION
**Pull request by fabian-guerra**
Since we branch `boba` not long ago and there are few changes in `master` we decided to merge the branch instead of cherry-picking individual changes.

### CODE REVIEW

**Comment by 1ec5** [comment]
We’re going to need a merge after 3.7.4 is released, and heatmaps (#11046) have yet to land in master, so perhaps we should hold off for a couple days?

**Comment by fabian-guerra** [comment]
@1ec5 makes sense let's wait until https://github.com/mapbox/mapbox-gl-native/pull/11046 lands.

**Comment by bsudekum** [comment]
@fabian-guerra can we cherry pick https://github.com/mapbox/mapbox-gl-native/pull/11197 onto this PR?

**Comment by fabian-guerra** [comment]
@bsudekum sure. I will redo this merge after https://github.com/mapbox/mapbox-gl-native/pull/11197 lands.

File: platform/android/scripts/exclude-activity-gen.json [hunk 1]
```diff
@@ -27,11 +27,12 @@
   "QueryRenderedFeaturesBoxHighlightActivity",
   "MultiMapActivity",
   "MapInDialogActivity",
+  "SimpleMapActivity",
```
Comment (tobrun) [inline platform/android/scripts/exclude-activity-gen.json]: this line seems to be duplicated (also found at the bottom of the file).

File: platform/macos/CHANGELOG.md [hunk 2]
```diff
@@ -4,20 +4,24 @@
...
+* Added the `MGLTileSourceOptionTileCoordinateBounds` option to create an `MGLTileSource` that only supplies tiles within a specific geographic bounding box. ([#11141](https://github.com/mapbox/mapbox-gl-native/pull/11141))
 * Fixed an issue preventing a dynamically-added `MGLRasterStyleLayer` from drawing until the map pans. ([#10270](https://github.com/mapbox/mapbox-gl-native/pull/10270))
 * Fixed an issue preventing `MGLImageSource`s from drawing on the map when the map is zoomed in and tilted. ([#10677](https://github.com/mapbox/mapbox-gl-native/pull/10677))
 * Improved the sharpness of raster tiles on Retina displays. ([#10984](https://github.com/mapbox/mapbox-gl-native/pull/10984))
 * Fixed a crash parsing a malformed style. ([#11001](https://github.com/mapbox/mapbox-gl-native/pull/11001))
+* Added the `MGLTileSourceOptionTileCoordinateBounds` option to create an `MGLTileSource` that only supplies tiles within a specific geographic bounding box. ([#11141](https://github.com/mapbox/mapbox-gl-native/pull/11141))
```
Comment (friedbunny) [inline platform/macos/CHANGELOG.md]: This is a duplicate of an entry being added a few lines above.

File: platform/ios/CHANGELOG.md [hunk 3]
```diff
@@ -34,15 +35,19 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ### Map snapshots
 
 * Fixed a memory leak that occurred when creating a map snapshot. ([#10585](https://github.com/mapbox/mapbox-gl-native/pull/10585))
-* Fixed an issue that caused `MGLMapSnapshotter.pointForCoordinate` to return an incorrect value. ([#11035](https://github.com/mapbox/mapbox-gl-native/pull/11035))
 
-### Other changes
+## Other changes
```
Comment (1ec5) [inline platform/ios/CHANGELOG.md]: This heading should be level 3, as before.

File: platform/ios/CHANGELOG.md [hunk 4]
```diff
@@ -34,15 +35,19 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
...
 
 * Feature querying results now account for the `MGLSymbolStyleLayer.circleStrokeWidth` property. ([#10897](https://github.com/mapbox/mapbox-gl-native/pull/10897))
 * Fixed an issue preventing labels from being transliterated when VoiceOver was enabled on iOS 10._x_ and below. ([#10881](https://github.com/mapbox/mapbox-gl-native/pull/10881))
 * Labels are now transliterated from more languages when VoiceOver is enabled. ([#10881](https://github.com/mapbox/mapbox-gl-native/pull/10881))
 * Long-pressing the attribution button causes the SDK’s version number to be displayed in the action sheet that appears. ([#10650](https://github.com/mapbox/mapbox-gl-native/pull/10650))
 
+## 3.7.4 - February 12, 2018
+
+* Added the `MGLTileSourceOptionTileCoordinateBounds` option to create an `MGLTileSource` that only supplies tiles within a specific geographic bounding box. ([#11141](https://github.com/mapbox/mapbox-gl-native/pull/11141))
```
Comment (1ec5) [inline platform/ios/CHANGELOG.md]: Remove the identical line from the 4.0.0 release notes.

File: platform/macos/CHANGELOG.md (hunk 2 above)
Comment (1ec5) [inline platform/macos/CHANGELOG.md]: This is duplicated.

File: platform/macos/CHANGELOG.md [hunk 5]
```diff
@@ -28,6 +32,7 @@
 
 * Added Danish and Hebrew localizations. ([#10967](https://github.com/mapbox/mapbox-gl-native/pull/10967), [#11136](https://github.com/mapbox/mapbox-gl-native/pull/11134))
 * Feature querying results now account for the `MGLSymbolStyleLayer.circleStrokeWidth` property. ([#10897](https://github.com/mapbox/mapbox-gl-native/pull/10897))
+* Added a Hebrew localization. ([#10967](https://github.com/mapbox/mapbox-gl-native/pull/10967))
```
Comment (1ec5) [inline platform/macos/CHANGELOG.md]: This is redundant to the first item in this list.

File: platform/ios/CHANGELOG.md (hunk 3 above)
Comment (friedbunny) [inline platform/ios/CHANGELOG.md]: This needs the third `#`, still. 🧐

File: platform/ios/CHANGELOG.md (hunk 4 above)
Comment (fabian-guerra) [reply platform/ios/CHANGELOG.md]: @1ec5 I saw this line duplicated in our macOS changelog but here.

**Pull request merged**